datalad/distribution/update.py

```diff
@@ -154,13 +183,17 @@ def _update_repo(ds, remote, merge, fetch_all, reobtain_data):
         if reobtain_data:
             # get all annexed files that have data present
             lgr.info('Recording file content availability to re-obtain update files later on')
-            reobtain_data = repo.get_annexed_files(with_content_only=True)
+            reobtain_data = \
+                [opj(ds.path, p)
```

Comment:
because we test later on if that thing still exists after upgrade? I don't think it represents a bottleneck of the entire operation in any case.